examples/soda_outgassing.jl

```diff
@@ -0,0 +1,286 @@
+# # Nutrients, plankton, bacteria, detritus
+#
+# This example illustrates how to use ClimaOceanBiogeochemistry's
+# `CarbonAlkalinityNutrients` model in a single column context.
+
+using ClimaOceanBiogeochemistry.CarbonSystemSolvers.UniversalRobustCarbonSolver: UniversalRobustCarbonSystem
+using ClimaOceanBiogeochemistry.CarbonSystemSolvers: CarbonChemistryCoefficients
+
+using Oceananigans
+using Oceananigans.Units
+
+using Printf
+using CairoMakie
+
+#
+# We are going to simulate two bottles of soda, one opened and left in the fridge
+#   the other opened and left on the counter to go flat.
+#
+
+# Build function for CO₂ flux calculation. Dissolved CO₂ in the soda will exchange with the overlying atmosphere
+# These are some coefficients and constants that we'll use in the calculation
+Base.@kwdef struct Pᶠˡᵘˣᶜᵒ²{FT}
+    surface_wind_speed   :: FT = 10. # ms⁻¹
+    atmospheric_pCO₂     :: FT = 280e-6 # atm
+    exchange_coefficient :: FT = 0.337 # cm hr⁻¹
+    salinity             :: FT = 35.0 # psu
+    alkalinity           :: FT = 2.35e-3 # mol kg⁻¹
+    silicate_conc        :: FT = 0e-6 # mol kg⁻¹
+    phosphate_conc       :: FT = 0e-6 # mol kg⁻¹
+    initial_pH_guess     :: FT = 8.0
+    reference_density    :: FT = 1000. # kg m⁻³
+end
+
+"""
+    compute_co₂_flux!(simulation)
+    
+Returns the tendency due to CO₂ flux using the piston velocity 
+formulation of Wanninkhof (1992) and the solubility/activity of 
+CO₂ that depends on temperature, etc.
+"""
+@inline function compute_co₂_flux!(simulation)
+# conversion factor from cm/hr to m/s
+    cmhr⁻¹_per_ms⁻¹ = 1 / 3.6e5
+
+# get coefficients from Pᶠˡᵘˣᶜᵒ² struct
+# I really want the option to take these from the model
+    (; surface_wind_speed, 
+       atmospheric_pCO₂, 
+       exchange_coefficient, 
+       salinity,
+       alkalinity,
+       silicate_conc, 
+       phosphate_conc,
+       initial_pH_guess, 
+       reference_density,   
+       ) = Pᶠˡᵘˣᶜᵒ²()
+
+    U₁₀      = surface_wind_speed  
+    pCO₂ᵃᵗᵐ  = atmospheric_pCO₂    
+    Kʷₐᵥₑ    = exchange_coefficient
+    Sᴬ       = salinity
+    Aᵀ       = alkalinity
+    Siᵀ      = silicate_conc   
+    Pᵀ       = phosphate_conc   
+    pH       = initial_pH_guess
+    ρʳᵉᶠ     = reference_density 
+    co₂_flux = simulation.model.tracers.DIC.boundary_conditions.top.condition
+
+    # applied pressure in atm (i.e. Pˢᵘʳᶠ-Pᵃᵗᵐ) 
+    # Positive when the can is sealed, then zero when the can is opens
+    # On average, the 12 ounce soda cans sold in the US tend to have a pressure of roughly 120 kPa when canned at 4 °C
+    if iteration(simulation) <= 1
+        Δpᵦₐᵣ   = 0.2
+    else
+        # *pssshhhht* the bottle is opened
+        Δpᵦₐᵣ   = 0.0
+    end
+
+    # access model fields
+    Θᶜ = simulation.model.tracers.T[
+        simulation.model.grid.Nx,
+        simulation.model.grid.Ny,
+        simulation.model.grid.Nz
+        ]
```

Comment:
```suggestion
    # access model fields
   Θᶜ = simulation.model.tracers.T[1, 1, Nz]
```

and get `Nz = size(grid, 3)` somewhere above. This is a single column model right? Not sure if the x, y indices have to be Nx or 1.